test/Groups/Permutations.jl

```diff
@@ -427,3 +428,22 @@ end
   H,iso = smaller_degree_permutation_representation(G)
   @test degree(H)<degree(G)
 end
+
+@testset "fixed_points" begin
+  # Trivial group (only identity) -- all points fixed
+  G = group((), 1:5)
+  @test sort(fixed_points(G)) == [1, 2, 3, 4, 5]
+
+  # Full symmetric group -- no points fixed
+  G = symmetric_group(5)
+  @test fixed_points(G) == Int[]
+
+  # Subgroup generated by (1,2); 3, 4, 5 are fixed
+  G = group(@perm(1,2), 1:5)
```

Comment:
```suggestion
  G = group(@perm((1,2)), 1:5)
```
I think this should do what you wanna do here. But since I type it on my phone, I haven't checked it myself